Source/WarpX.cpp

```diff
@@ -967,15 +1015,26 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
         Efield_aux[lev][0].reset( new MultiFab(nba,dm,ncomps,ngE));
         Efield_aux[lev][1].reset( new MultiFab(nba,dm,ncomps,ngE));
         Efield_aux[lev][2].reset( new MultiFab(nba,dm,ncomps,ngE));
+#ifdef WARPX_MAG_LLG
+        Mfield_aux[lev][0].reset( new MultiFab(nba,dm,3     ,ngE));
+        Mfield_aux[lev][1].reset( new MultiFab(nba,dm,3     ,ngE));
+        Mfield_aux[lev][2].reset( new MultiFab(nba,dm,3     ,ngE));
+
+        H_biasfield_aux[lev][0].reset( new MultiFab(nba,dm,ncomps,ngE));
+        H_biasfield_aux[lev][1].reset( new MultiFab(nba,dm,ncomps,ngE));
+        H_biasfield_aux[lev][2].reset( new MultiFab(nba,dm,ncomps,ngE));
+#endif
```

Comment:
all this is in ifdef PSATD. May not be required?

Reply:
hmmm. I have hard time recognizing the end of #ifdef WARPX_USE_PSATD in line 968. Maybe you can help me with it?

Reply:
PSATD does not cover this part per our discussion. I will close this conversation.